generators/client/templates/angular/src/main/webapp/content/scss/vendor.scss.ejs

```diff
@@ -23,9 +23,16 @@ put Sass variables here:
 eg $input-color: red;
 ****************************/
 // Override Boostrap variables
+<%_ if (clientTheme === 'none') { _%>
 @import "bootstrap-variables";
```

Comment:
we shouldn't make this conditional, as it would be useful to customize selected theme. Instead, we should make it's content conditional